### ISSUE TITLE
Change time of node 6 or 7

### ISSUE THREAD

**Issue by jeromekelleher** [daikitag/tstrait-paper#13]
It's potentially distracting to have nodes 6 and 7 at exactly the same time.  Change one of them to be slightly more or less

Related to #10

**Comment by jeromekelleher** [comment]
Here's the code to partly fix this and #10:
```python
...
for i in range(1,5):
    node.add_row(time=i + 1)
...
mut_labels = {0: "A→T", 1:"T→A"}
alphabet = "abcdefghijkl"
node_labels = {u: alphabet[u] for u in range(len(table.nodes))}
tree = table.tree_sequence().first()
tree.draw_svg(size=(300, 300),
              node_labels=node_labels,
              mutation_labels=mut_labels, 
              path="tree_fig.svg")
```

**Comment by daikitag** [comment]
Resolved in https://github.com/daikitag/tstrait-paper/pull/16
